pkg/cmd/cli/describe/chaindescriber.go

```diff
@@ -92,6 +93,15 @@ func (d *ChainDescriber) Describe(ist *imageapi.ImageStreamTag, includeInputImag
 		return "", NotFoundErr(fmt.Sprintf("%q", ist.Name))
 	}
 
+	markers := buildanalysis.FindCircularBuilds(g, osgraph.DefaultNamer)
+	if len(markers) > 0 {
+		for _, marker := range markers {
+			if strings.Contains(marker.Message, ist.Name) {
+				return fmt.Sprintf("Exiting examination of %s. %s", ist.Name, marker.Message), nil
```

Comment:
With the code in its current form:
```
Exiting examination of ruby-22-centos7:latest. Cycle detected in build configurations: BuildConfig|example/ruby-22-centos7 -> ImageStreamTag|example/ruby-hello-world:latest -> BuildConfig|example/ruby-hello-world -> ImageStreamTag|example/ruby-something-else:latest -> BuildConfig|example/ruby-something-else -> ImageStreamTag|example/ruby-22-centos7:latest -> BuildConfig|example/ruby-22-centos7
```

If we just went with the `marker.Message`:
```
Cycle detected in build configurations: BuildConfig|example/ruby-22-centos7 -> ImageStreamTag|example/ruby-hello-world:latest -> BuildConfig|example/ruby-hello-world -> ImageStreamTag|example/ruby-something-else:latest -> BuildConfig|example/ruby-something-else -> ImageStreamTag|example/ruby-22-centos7:latest -> BuildConfig|example/ruby-22-centos7
```
When the code did *not* employ the circular build detection, it produced the following when a cyclic dependency existed between 3 build config / image stream tuples:
```
		istag/ruby-22-centos7:latest
			bc/ruby-22-centos7
				istag/ruby-hello-world:latest
					bc/ruby-hello-world
						istag/ruby-something-else:latest
							bc/ruby-something-else
```